src/util/pointer_predicates.h

```diff
@@ -33,7 +33,6 @@ exprt good_pointer_def(const exprt &pointer, const namespacet &);
 exprt null_object(const exprt &pointer);
 exprt null_pointer(const exprt &pointer);
 exprt integer_address(const exprt &pointer);
-exprt is_invalid_pointer(const exprt &pointer);
```

Comment:
I guess that's ok, though generally we should first mark such functions as deprecated.